frontend/src/pages/LoginPage.js

```diff
@@ -1,7 +1,85 @@
+import Head from '../components/Head';
+import Footer from '../components/Footer';
+import Navlink from '../components/Navlink';
+import FormTemplate from '../components/FormTemplate';
+import { useState, useEffect } from 'react';
+import { Form, Container, Row, Button } from 'react-bootstrap';
+
```

Comment:
as general practice we divide different type of imports by space

example
```
// package imports here
import { useState, useEffect } from 'react';
import { Form, Container, Row, Button } from 'react-bootstrap';

// react component imports here
import Head from '../components/Head';
import Footer from '../components/Footer';
import Navlink from '../components/Navlink';
import FormTemplate from '../components/FormTemplate';

// other file imports here
```

please update. no need to include the comments

Reply:
Got it Sir. I'll change it right now.